src/FuncHW/ListExtensions.cs

```diff
@@ -12,5 +13,10 @@ public static class ListExtensions
         //add extension for List CountElements: receives Func, returns the number of elements that match the condition
 
         //add extension for List SelectWhereNot: receives Func, returns the list of elements that don't match the condition
+
+        public static T GetLast<T>(this List<T> values, Func<T, bool> func)
+        {
+            return values.Last<T>(func);
```

Comment:
please, don't use Linq here. the point is to create extension like in Linq. there is no reason to write extension if you use Linq

Reply:
Changed in commit e1328a348a37818a9cd56ba204e5785a20e44e4d